content/en/blog/_posts/2021-12-16-prevent-persistentvolume-leaks-when-deleting-out-of-order.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: blog
 title: "Kubernetes 1.23 Prevent PersistentVolume leaks when deleting out of order"
-date: 2021-12-15T10:00:00-08:00
+date: 2021-12-16
```

Comment:
This article already published as scheduled.

I don't think we need to make this change.

/hold